### ISSUE TITLE
No Widget

### ISSUE THREAD

**Issue by maverick74** [d86leader/dynamic_workspaces#1]
This is probably more of an "Interface" problem (where the "interface" is what connects the chair to the keyboard), but i can't seem to find any widget to get it working...

What am i missing?

**Comment by d86leader** [comment]
Hello, not sure what you're talking about. This isn't an applet or a widget or an app, it's just a kwin script. You install it through the KDE store or by following the steps in the readme, and it should start working without any widgets or buttons. If it's not working, then there is an issue, and it's got to be debugged.

The closest to a widget you can find is as follows: open KDE system settings -> window management -> kwin scripts. You should see one named "dynamic workspaces", and from here you can disable, re-enable or delete it.

**Comment by maverick74** [comment]
Hi.

Ok...then maybe I'm not getting it right: when you say workspace, aren't you talking about virtual desktops?

**Comment by d86leader** [comment]
Yes I do mean those. This script enhances the virtual desktops built into plasma5 by providing the additional functionality of dynamically creating them.
Do the KDE virtual desktops work for you otherwise? Try adding the "Pager" applet to a panel and playing with that: create, switch and delete desktops. Then install this kwin script, relogin for a good measure, and try moving windows around your virtual desktops.
You can then still use the "Pager" applet as an indicator of desktops. Actually what you see on the script page in the opendesktop store are the screenshots of the "Pager" applet in different states.
